code/_core/obj/item/clothing/back/storage/backpack/explorer_pack.dm

```diff
@@ -1,106 +1,110 @@
-obj/item/clothing/back/storage/backpack/explorer

```

Comment:
Did you edit this file? It seems to have an issue with EOL stuff again.

Reply:
I did, however, it had no issues when I was compiling.

Reply:
Oh god it just deleted and rewrote everything wtf.